src/Schema/Directives/FindDirective.php

```diff
@@ -35,9 +35,12 @@ public static function definition(): string
     public function resolveField(FieldValue $fieldValue): callable
     {
         return function ($root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo): ?Model {
+            /** @var \Illuminate\Database\Eloquent\Builder<Model> $builder */
```

Comment:
I feel like this should not be necessary. Please revert and let us explore other options